### ISSUE TITLE
Do you tune YOLOX provided in ByteTrack with other tracker?

### ISSUE THREAD

**Issue by wenyuqing** [megvii-research/MOTRv2#36]
I previously thought that the yolox weight is from training yolox with ByteTrack tracker on DanceTrack training set, which according to the results in this link you provided in the paper: [https://github.com/DanceTrack/DanceTrack] as you provided in the paper ( 47.7 HOTA)
![image](https://user-images.githubusercontent.com/48059861/227089891-5039f662-cd69-4586-b410-d1c90408e20b.png)

But it seems that you use OC-SORT tracker rather than ByteTrack tracker?  Do you mean you use the model that have 55.1 HOTA results on DanceTrack test set?
![image](https://user-images.githubusercontent.com/48059861/227090295-270da165-9e46-4ed3-aa50-f3614c8a9b24.png)

Have you tried other yolox weights such as the yolox pretrained with ByterTrack tracker(47.7 HOTA)? Will it influence the overall performance a lot?

Looking forward to your reply, thanks!

_Originally posted by @wenyuqing in https://github.com/megvii-research/MOTRv2/issues/34#issuecomment-1480542920_
